conda.yml

```diff
@@ -0,0 +1,8 @@
+name: variant-remapping
+channels:
+  - bioconda
+  - defaults
+dependencies:
+  - bedtools=2.29.2
+  - bowtie2=2.4.1
+  - samtools=1.4.1-0 
```

Comment:
By the way, could you also add instructions to install the dependencies and to run this simple Nextflow workflow?

Reply:
I've not used conda before but have usually heard good things about it. I though that would be a nice way to manage the multiple dependencies that the pipeline has. It will also help if we want to run the pipeline in travis

Reply:
Sure. Let's try it and see how it works. I'm sure if we manage it properly it should also behave accordingly.